Tiltfile

```diff
@@ -0,0 +1,14 @@
+# Deploy: tell Tilt what YAML to deploy
+k8s_yaml('deploy/kubernetes-manifests.yaml')
+
+custom_build(
+  'deislabs/smi-adapter-istio',
```

Comment:
Does this need to change to the user defined container?

Reply:
Yes, there are instructions in the CONTRIBUTING.md that address that.